src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/telemetry/telemetry-configuration/telemetry-configuration.component.html

```diff
@@ -0,0 +1,207 @@
+<cd-loading-panel *ngIf="loading && !error"
+                  i18n>Loading configuration...</cd-loading-panel>
+<cd-alert-panel type="error"
+                *ngIf="loading && error"
+                i18n>The configuration could not be loaded.</cd-alert-panel>
+
+<div class="cd-col-form"
+     *ngIf="!loading && !error">
```

Comment:
I'll have a look at it, thanks!

Reply:
> What happens if loading == false && error == true? Why not breaking down ngIf conditions?:

@epuertat
The case won't happen because ONLY in the success case loading will be set to false. Otherwise we received an error and error will be set to true.
We do have the same implementation on other modules as well, e.g.: https://github.com/ceph/ceph/blob/master/src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/mgr-modules/mgr-module-form/mgr-module-form.component.html#L1
May I suggest to leave the code as it is for this PR and create an issue to address your comment in a more generic fashion? Would you be fine with that?

Reply:
https://tracker.ceph.com/issues/45146